### DISCUSSION TITLE
Is it possible to have more than 1 Twilio (dynamic) configuration?

### DISCUSSION THREAD

**Discussion by seyaobey-dev** [nhost/nhost#1593] [Q&A]
Hello Nhost team. 

Is it possible to have more than 1 Twilio configuration for sms authentication (and swap them dynamically)? 

I'm using Nhost with my react-native app. I was having an issue where, in certain countries, the sms OTP code for authentication was not been received. 

After having consulted with Twilio support, it turns out that I may need different twilio numbers, depending on the country. 

Certain countries require only local numbers, other countries only international numbers, others only alphanumeric sender ID...etc 

So only 1 Messaging Service ID may not be enough to cover these scenarios

**Comment by elitan** [comment]
When you configure Twilio at Nhost, you can set a Messagagin Service ID. 

From [Twilios Docs](https://www.twilio.com/docs/sms/send-messages#messaging-services):

> You can also enter a Messaging Service SID as the From parameter value when you send an SMS message. When you use a Messaging Service SID, Twilio will automatically send your message using one of the phone numbers in your service pool.

This should solve your problem. From my understanding, Twilio will decide what phone number to pick depending on who you send the SMS to.

More docs: https://www.twilio.com/docs/messaging/services

Do you think that solves this problem?
